### DISCUSSION TITLE
Power Shovel - Boulders - Diamonds

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2287] [Ideas]
![image](https://user-images.githubusercontent.com/108110065/221391100-da775f21-d4fb-4f73-a5d7-de80cdfbd03c.png)

Hello, below is a proposal about a mechanic that would tie power shovel, boulders and diamonds together as 1 part of gameplay.
The aim is to start a discussion on how boulders would work and how to mine diamonds. Have not found any discussions around this and I am very much looking forward to being able to move plots, trees etc. around to make my farm pretty! So here we go..

**TLDR:** 
_Players woud be crafting GOLDEN PICKAXES (Ingredients below), players would hit the boulder with the golden pickaxe and an array of resources would be yielded from it. The idea is that there is a small chance a player would receive 1 diamond, but if unsuccessfull, it would give back the player 80% of its resources that they used to craft the golden pickaxe. Power shovel once unlocked is with the player forever._

**MORE DETAILS:**
Essentially, this becomes a burn mechanic, a player uses 100 resources to try and get a diamond, if not successfull the player receives back 80 resources and can try again the next day (Boulder to have 24-36h cooldown). 5 diamonds required for a power shovel: a player can be lucky and get 2-3 diamonds quicker than usual but may get the last 2 longer than expected, the reverse is also true.

This in my opinion is intended to be a high level part of the game with relatively high costs BUT is forgiving due to part of the resources being handed back.

Diamond could also later be used for other special mints, buildings or items. It would become a resource similar to gold stone etc. able to be traded on the market. It would be VERY rare, much more than gold.

**Base costs along with proposed Golden Pickaxe and Diamond Shovel costs:**

![image](https://user-images.githubusercontent.com/108110065/221391715-3547b901-04a9-46e4-83b6-9ad5c5b3774f.png)

^ Diamond has 2 costs.. 12.236 is the cost of a golden pickaxe, but we will not realistically receive 1 diamond by using the golden pickaxe once... I put diamond as a 7.5% chance of being yielded which translates to 13.3 golden pickaxes required on average to yield a diamond. Because of this, 13.3~ golden pickaxes cost 163 SFL, which is the realistic value of diamond (at base costs).

Below are tables showing possible returns of resources if no diamond found, 3 examples shown. Note how 1 table has no wood, technically a boulder does not contain wood judging by the art, so maybe it should not drop any?

![image](https://user-images.githubusercontent.com/108110065/221391180-6ccf2eca-fa2c-41d5-bc35-781c33cfd603.png)

We can also calculate days ON AVERAGE required to find enough diamonds to crafts a powershovel. at 5 diamonds needed currently, 13.3 hits on average to find 1, 13.3*5 = ~67 days for a player to craft it. Can be changed to be more/less...

![image](https://user-images.githubusercontent.com/108110065/221392356-0ef797f9-f0d7-425f-9dc9-4badf7050756.png)

^ Cost of Power Shovel is 815.7 sfl = (cost of 1 golden pickaxe * hits needed for 1 diamond) * diamonds needed for powershovel....But because we are getting 80% of resources back upon missing the diamond, the real cost of a power shovel is 20% of that which translates to 163SFL. If anyone finds a mistake let me know :D

_**ALL VALUES, CHANCES AND EVERYTHING CAN BE TWEAKED**
We can make it easier, or harder :) I prefer to make it harder, personally. Game needs an ingame sink and this would in theory make wood/minerals more 'rare'._

### **PROS**:
- Provide a gameplay burn mechanic, on the main island itself
- Increase rarity of resources as players are receiving back less than what they are putting in
- Sort out unfinished gameplay aspects ingame currently, power shovel, boulder and diamond.
- With farm items affecting x crops/trees etc. in their surrounding, it will provide incentives for players to unlock the power shovel to group trees/plots together
- Mid/High end players can have something to do rather than just farm, mine and sell, which is something I am finding myself doing..

### **CONS**:
- Too much RNG? (I argue that because it is over a long stretch of time to get 5 diamonds, averaged out it should be similar for everyone. Some might get diamonds quicker at first but slower later, and viceversa)
- Too high of a cost?
- This proposal is garbage?

Thanks for reading this far, if you hate the idea I would strongly suggest to provide alternatives along with the feedback if possible.

A quote from Adam regarding Power Shovel:
![Screenshot_20230226-221136](https://user-images.githubusercontent.com/108110065/221407050-aa78429e-3435-4813-877c-f9f37c4a4598.png)

I just want to see these features implemented if community likes them, I am doing my small part to get the discussions going and a bit of ground work done for the devs. Thank you =)

**Comment by LandLadyFarmer** [comment]
To late at night to read it thorough (will do) but for now i just have some thoughts.
It's not a bad idea to have something else as a sink because when we don't have a mint like iron idol i believe treasure island will be less used specially if there is no need to level up, maybe they with great boost still use it most people wont.

Adam said power shovel will be reusable so when we crafted power shovel will diamonds in an near future be needed for something else so majority keeps mine for diamonds after they crafted shovel or will they just be a resource that sits in you inventory so most people will stop mine for diamonds, who knows!!.. ( i know you can't answer that ) I'm just thinking loud.
I know i will still collect diamonds but i don't believe average player wont and if its to hard and expensive I'm not sure if they even 
would go for shovel..
I believe most people are dying for be able to move nodes but if it gets to hard and expensive it can be something that a lot of people start complaining about and becomes a negative experience...

I will definitely read through this thorough and give it some time to sink in, I will be back and give my opinion on this but for now i have to sleep on this.. It's early morning so it's time to give up this and give it some thought..
interesting proposal tho got me thinking!!!

**Reply by TheDcol** [reply]
Thanks mate :)

To answer one of your questions, i believe diamonds would be needed for other things in the future

**Comment by raoraoxu** [comment]
If so many resources are needed for landscaping, I will give up.
The Decorations & Landscaping system under development will not work.
There will still be objective income if the land is not landscaping.
Why does the team have some bad ideas when the game is on the right track?

**Reply by TheDcol** [reply]
Is 163sfl too high of a cost for the power shovel in your opinion?

Also, keep in mind as stated above: "ALL VALUES, CHANCES AND EVERYTHING CAN BE TWEAKED"

The costs/values can be changed, the mechanic itself is more of the feedback i am after cheers appreciate your comment and your self-liking reacts 👍

Also keep in mind, Adam literally said this on that landscaping proposal:

![Screenshot_20230226-221136](https://user-images.githubusercontent.com/108110065/221407080-ab2563a1-af36-4c50-a9ff-c6f291c7a38c.png)

**Reply by raoraoxu** [reply]
I imagine that low-cost landscaping is to attract nurturance players or female players, and they build beautiful land. If the team wants to use landscaping to consume resources, it needs to consider how to let a large number of players dig diamonds actively at high cost.
Farming Equipment is not to attract new players. Is it the short-term goal of new players?
If new players use agricultural equipment not landscaping , will senior players also do so because of the high cost?
Please give priority to new players. This is the need of the community, but old players are also important.
Please be careful，thank you for your reply

**Reply by TheDcol** [reply]
I would not generalise and say female players would be attracted to landscaping.. they play this game already without it.. 

Farming equipment is for everyone, new and old players.

Aeon covered it well here https://github.com/sunflower-land/sunflower-land/discussions/2287#discussioncomment-5114502

**Reply by raoraoxu** [reply]
Well, Aeon convinced me temporarily

**Reply by rasheed19871** [reply]
If moving my node and tree cost diamonds really I prefer it to stay the way it is, diamond is good and I like it but for me I will not use it

**Reply by TheDcol** [reply]
Yup, you can leave your land as is 😃

**Comment by Aeon-LlamaMod** [comment]
Great proposal as usual. I believe the power shovel should be a high cost. This is end game content that most users need to work for. The boulders will have a use then and players have something to work for. 

The devs have already said that land aka space is already a resource and thus the ability to move things around opens up more space or can be seen as additional space not have equivalent to an expansion. Thus if users want that extra space or ability to move things around i think there has to be a step cost. 

Thoughts on diamonds being tradeable? Or shortly after their release? The issue is some users may not care to make the power shovel and just want to sell to those that do. This would open that up. If there is a % chance to get a diamond in a dig then i cant see diamonds being good to use to mint nfts because then we come back to the iron idol issue where people would be upset its % base luck to mint and not dedication.

**Reply by TheDcol** [reply]
Thanks mate,

Open to diamonds being tradeable, it would be a resource like gold

Diamond could later be used for special helios sfts, so they do not run out and are available to everyone. They would only be restricted to time

**Comment by brian-man** [comment]
At least half a dozen VIP, HODL and Tractor players have explicitly stated in discord multiple times that the new RNG direction of the game is [a] making them extract more often, [b] nudging them to disengage, and [c] likely will make them exit the game in 2023.

In light of this, why does the team keep introducing more RNG into the game?

**Reply by TheDcol** [reply]
Rng is present in many aspects of the game, players do not like bad rng. Green amulet proc, gold rush proc, extra yield proc, tough tree proc... these are all rng aspects.

Land expansion rng is something else which i also do not love, you roll one chance and you are stuck with it.

A lot of people on discord will complain about anything that makes them spend resources/money or cut away from their profits.. i do not think they care too much about the game and just want to extract/make profit. Naturally, rhey would be against burn mechanics. These are the same people who will complain about the lack of ingame burn resulting in resource inflation.

The alternative is to make golden pickaxe cost 200 sfl, it yields diamonds 100% of the time and boulders have a 2 week cooldown. Imo this is the lazy way of doing it and the wrong way, as there should be a burn for a high value item.

Also I am not part of the team, just a community member

**Reply by brian-man** [reply]
> A lot of people on discord will complain about anything that makes them spend resources/money or cut away from their profits

It's not helpful to conflate this with RNG concerns.

> The alternative is to make golden pickaxe cost 200 sfl, it yields diamonds 100% of the time and boulders have a 2 week cooldown.

Okay.  How about this.  Why not introduce both options and let the players (individually) decide?  This puts appropriate pressure on the team to ensure that the RNG path is compelling enough that people choose that path.

> Also I am not part of the team, just a community member

I'm aware. :)  But you have closer discussions with the decision-makers and you're presenting a proposal to introduce more RNG.

**Reply by TheDcol** [reply]
I don't view it as bad rng especially if it is something not time limited and not supply limited. It is also not a one off rng chance but over course of time

Adding both options is interesting, maybe a more expensive than average Golden Drill..

Golden Drill - guaranteed 1 diamond yield - 250sfl
Golden pickaxe - same chances as above - 165sfl

**Comment by brian-man** [comment]
### Additional concern
IMO, being able to rearrange your nodes without ridiculous costs or waiting periods is critical for making the farm equipment (sprinklers, etc.) features broadly appealing and available.  As such, I don't think it makes sense to tie significant costs to the first round of landscaping. 

### Counter-proposal
1. Introduce a cheaper sibling -- Mechanical Shovel -- at Helios for a reasonable price.
2. Mechanical Shovel would be land-bound (not withdrawable).
3. Mechanical Shovel could either be very cheap and one-day use OR medium priced and available for one-day use with a 30d cooldown.
4. In contrast, Power Shovel would have either no cooldown or a much smaller cooldown (1d? 7d?).

**Reply by TheDcol** [reply]
Sprinklers and other farm equipment will still grant quite a bit of extra yield without players moving things around. The power shovel would be a lucrative long term goal for a farmer, which the game lacks atm.

Adding cheaper siblings would remove the need to chase the power shovel, move everything in 1 hour and you are set for a few months.

But good suggestions, i also though of making it a tool for each resource to move them but might have been overly complicated

**Comment by Nimr1** [comment]
I like this solution to Power shovel and the Boulders. 

Think it could tie in with my new Stone Haven concept idea I wrote recently.
